src/locale/locales/de/messages.po

```diff
@@ -1180,113 +1146,106 @@ msgstr "Neue Benachrichtigungen laden"
 msgid "Loading..."
-msgstr "Laden..."
+msgstr "Wird geladen..."
 
 #: src/view/com/modals/ServerInput.tsx:50
 msgid "Local dev server"
-msgstr "Lokaler Entwicklungs-Server"
+msgstr "Lokaler Entwicklungsserver"
 
 #: src/view/screens/Moderation.tsx:136
 msgid "Logged-out visibility"
-msgstr "Ausgeloggte Sichtbarkeit"
+msgstr ""
 
 #: src/view/com/auth/login/ChooseAccountForm.tsx:133
 msgid "Login to account that is not listed"
 msgstr "Anmeldung bei einem Konto, das nicht aufgelistet ist"
 
-#: src/view/screens/ProfileFeed.tsx:472
-#~ msgid "Looks like this feed is only available to users with a Bluesky account. Please sign up or sign in to view this feed!"
-#~ msgstr "Es sieht so aus, als ob dieser Feed nur für Benutzer mit einem Bluesky-Konto verfügbar ist. Bitte melde dich an oder registriere dich, um diesen Feed zu sehen!"
-
 #: src/view/com/modals/LinkWarning.tsx:63
 msgid "Make sure this is where you intend to go!"
-msgstr "Vergewissern Sie sich, dass Sie dorthin gehen wollen!"
+msgstr "Vergewissere dich, dass du auch wirklich dorthin gehen willst!"
 
 #: src/view/screens/Profile.tsx:163
 msgid "Media"
 msgstr "Medien"
 
 #: src/view/com/threadgate/WhoCanReply.tsx:139
 msgid "mentioned users"
-msgstr "genannte Nutzer"
+msgstr "erwähnte Benutzer"
 
 #: src/view/com/modals/Threadgate.tsx:93
 msgid "Mentioned users"
-msgstr "Genannte Nutzer"
+msgstr "Erwähnte Benutzer"
 
 #: src/view/screens/Search/Search.tsx:529
 msgid "Menu"
 msgstr "Menü"
 
 #: src/view/com/posts/FeedErrorMessage.tsx:194
 msgid "Message from server"
-msgstr "Meldung vom Server"
+msgstr "Nachricht vom Server"
 
-#: src/view/screens/Moderation.tsx:64
-#: src/view/screens/Settings.tsx:563
-#: src/view/shell/desktop/LeftNav.tsx:395
-#: src/view/shell/Drawer.tsx:509
+#: src/view/screens/Moderation.tsx:64 src/view/screens/Settings.tsx:563
+#: src/view/shell/desktop/LeftNav.tsx:395 src/view/shell/Drawer.tsx:509
 #: src/view/shell/Drawer.tsx:510
 msgid "Moderation"
 msgstr "Moderation"
 
 #: src/view/screens/Moderation.tsx:95
 msgid "Moderation lists"
-msgstr "Moderation Liste"
+msgstr "Moderationslisten"
 
 #: src/view/screens/ModerationModlists.tsx:58
 msgid "Moderation Lists"
-msgstr "Moderation Liste"
+msgstr "Moderationslisten"
 
 #: src/view/shell/desktop/Feeds.tsx:53
 msgid "More feeds"
-msgstr "Mehr Feeds "
+msgstr "Mehr Feeds"
 
 #: src/view/com/profile/ProfileHeader.tsx:548
-#: src/view/screens/ProfileFeed.tsx:360
-#: src/view/screens/ProfileList.tsx:583
+#: src/view/screens/ProfileFeed.tsx:360 src/view/screens/ProfileList.tsx:583
 msgid "More options"
 msgstr "Mehr Optionen"
 
 #: src/view/com/profile/ProfileHeader.tsx:370
 msgid "Mute Account"
-msgstr "Stummes Konto"
+msgstr "Konto stummschalten"
 
 #: src/view/screens/ProfileList.tsx:510
 msgid "Mute accounts"
-msgstr "Stummes Konto"
+msgstr "Konten stummschalten"
 
 #: src/view/screens/ProfileList.tsx:457
 msgid "Mute list"
-msgstr "Stumme Liste"
+msgstr ""
```

Comment:
That's difficult. "Stummschaltliste" (list that mutes [accounts]) sounds a bit clumsy, but it's better than the literal "Stummliste" (muted list)

Reply:
Isn't this used as the string for the action to mute a list? Then it should be `"Liste stummschalten"`.